service/src/main/java/bio/terra/pipelines/service/StatusService.java

```diff
@@ -20,11 +21,13 @@ public class StatusService extends BaseStatusService {
   public StatusService(
       NamedParameterJdbcTemplate jdbcTemplate,
       StatusCheckConfiguration configuration,
-      SamService samService) {
+      SamService samService,
+      WorkspaceService workspaceService) {
     super(configuration);
     this.jdbcTemplate = jdbcTemplate;
     registerStatusCheck("CloudSQL", this::databaseStatus);
     registerStatusCheck("Sam", samService::checkHealthApiSystemStatus);
+    registerStatusCheck("WorkspaceManager", workspaceService::checkHealthApiSystemStatus);
```

Comment:
looking at the history of SamService it looks like i was the one who added this function to that 🤔 .  I have no idea why I didnt add the other external clients in the same way.  I think i tried changing the `databaseStatus` to return a `Result` and i coudlnt get it to work so i gave up =/